From 6c73f28b6d87a2afd11974552a075bffec52347f Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Fri, 29 Nov 2013 20:57:46 +0000
Subject: Bug 719560 - Build failure with freetype 2.5.1

Apply patch from su-v that fixes the freetype include to
the madness devised and recommended by freetype.
---
diff --git a/app/text/gimpfont.c b/app/text/gimpfont.c
index 4045ca9..66c6e52 100644
--- a/app/text/gimpfont.c
+++ b/app/text/gimpfont.c
@@ -28,7 +28,9 @@
 
 #define PANGO_ENABLE_ENGINE  1   /* Argh */
 #include <pango/pango-ot.h>
-#include <freetype/tttables.h>
+
+#include <ft2build.h>
+#include FT_TRUETYPE_TABLES_H
 
 #include "text-types.h"
 
--
cgit v0.9.2
